flujo/recipes/factories.py

```
@@ -411,17 +411,60 @@
+    # 2) If still missing, try to locate the explicit validate step by name
     if checklist is None:
-        checklist = context.scratchpad.get("checklist")
+        for step in result.step_history:
+            try:
+                if getattr(step, "name", "") == "validate_step" and isinstance(
+                    step.output, Checklist
+                ):
+                    checklist = step.output
+                    break
+            except Exception:
+                continue
```

Ruff (0.12.2)

`try`-`except`-`continue` detected, consider logging the exception

(S112)